CONFIG.md

```diff
@@ -0,0 +1,419 @@
+You can configure Hazelcast Node.js Client declaratively (JSON) or programmatically (API).
+* Programmatic configuration
+* Declarative configuration (JSON file)
+
+# Programmatic Configuration
+You can configure your cluster programmatically. For this you can create a Config object, set/change its properties
```

Comment:
Term `configuring cluster` may be misleading.
This sentence from Hazelcast Reference manual can be better:
>For programmatic configuration of the Hazelcast Java Client, just instantiate a ClientConfig object and configure the desired aspects. An example is shown below.

Reply:
👍